### ISSUE TITLE
Working ? 

### ISSUE THREAD

**Issue by Adefx** [FabTangi/ttgo-ttn-ds18b20#1]
HI ! Im' quite interested into your project since i have plenty of TTGO T-beam v1.1 around. 
Did you succeed making the T-beam working with your code ? 

Thanks in advance

**Comment by FabTangi** [comment]
Hello, 
Yes it works with such a device https://fr.aliexpress.com/item/32915894264.html
I did not try with T-Beam as I don't need GPS

**Comment by Adefx** [comment]
I've just succeeded made it working with a T-Beam 1.1. 
of course GPS doesn't work but that not the point here.
do you have the TTN payload decoder around ? (I'm lazy :-)  )
[EDIT] The payload is in a Cayenne LPP format

**Comment by FabTangi** [comment]
Yes exactly, LPP was easier for some integrations (Domoticz for ex)

**Comment by Adefx** [comment]
Or Jeedom ;-)  
Concerning that i have some problem decoding the MQTT informations. But it's another subject. 
Is there anywhere we could exchange more easily then a Issue thread :-)  ? (discord/messenger)
